test/integration/smart_answer_flows/marriage_abroad_test.rb

```diff
@@ -1131,16 +1131,154 @@ def self.translations
     end
   end
 
-  context "ceremony in sweden, resident in sweden, partner other" do
+  context "ceremony in sweden," do
     setup do
       worldwide_api_has_organisations_for_location('sweden', read_fixture_file('worldwide/sweden_organisations.json'))
       add_response 'sweden'
-      add_response 'ceremony_country'
-      add_response 'partner_other'
-      add_response 'same_sex'
     end
-    should "go to cp or equivalent os outcome" do
-      assert_current_node :outcome_same_sex_civil_partnership
+    context "resident in uk," do
```

Comment:
It's worth noting that this style of nested contexts has been deprecated in favour of "flatter" tests. See https://github.com/alphagov/smart-answers/blob/master/doc/testing.md#deprecated-a-test-using-nested-contexts for more information.

I think the tests are fine in this case given that that's how they're structured in the rest of this file.